<launch>
  <include file="$(find uvc_camera)/launch/camera_node.launch" ns="camera0" >
    <arg name="device" default="/dev/video0" />
  </include>
</launch>
